cmd/root.go

```diff
@@ -87,17 +93,17 @@ func configureLogrusHooks() error {
 	}
 	// if env var not set, ignore
 	if sentryEnv.DSN == "" {
-		logrus.Debug("Sentry DSN not set. Skipping Sentry Configuration")
+		log.Debug("Sentry DSN not set. Skipping Sentry Configuration")
```

Comment:
why?

Reply:
Oh, I imported logrus twice (one called "log", one called "logrus") and just went with the log one